support-workers/src/typescript/model/billingPeriod.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod';
+
+export const billingPeriodSchema = z.union([
+	z.literal('Monthly'), //TODO: share this with support-frontend
```

Comment:
I've decided to do all the TODOs around sharing models in a follow up PR because it will probably involve some quite extensive changes